Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Oceananigans"
 uuid = "9e8cae18-63c1-5223-a75c-80ca9d6e9a09"
 authors = ["Climate Modeling Alliance and contributors"]
-version = "0.100.7"
+version = "0.101.0"
```

Comment:
How about going to v1 at some point?  😁  At least you'd have more space to distinguish breaking changes/new features/patches, with only two numbers available that becomes more complicated.

Reply:
#4911